sematic/plugins/building/docker_builder.py

```diff
@@ -39,6 +42,24 @@
 # TODO: bump to 0.1.0 when the build system is exposed to users
 _PLUGIN_VERSION = (0, 0, 1)
 
+_DOCKERFILE_BASE_TEMPLATE = """
+FROM {base_uri}
+WORKDIR /
+
+RUN which pip3 || apt update -y && apt install -y python3-pip
+RUN python3 -c "import distutils" || apt update -y && apt install --reinstall -y python3.9-distutils
```

Comment:
```suggestion
RUN PYTHON_MAJOR_MINOR=$(python3 -c "import sys; print(f'{sys.version_info.major}.{sys.version_info.minor}')")
RUN python3 -c "import distutils" || apt update -y && apt install --reinstall -y python$PYTHON_MAJOR_MINOR-distutils
```
in case the user's image is on something other than python 3.9